x-pack/plugins/task_manager/server/saved_objects/mappings.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsTypeMappingDefinition } from '@kbn/core/server';
+
+export const taskMappings: SavedObjectsTypeMappingDefinition = {
+  dynamic: false,
+  properties: {
+    taskType: {
+      type: 'keyword',
+    },
+    scheduledAt: {
+      type: 'date',
+    },
+    runAt: {
+      type: 'date',
+    },
+    // NO NEED TO BE INDEXED
+    // startedAt: {
+    //   type: 'date',
+    // },
+    retryAt: {
+      type: 'date',
+    },
+    enabled: {
+      type: 'boolean',
+    },
+    schedule: {
+      properties: {
+        interval: {
+          type: 'keyword',
+        },
+      },
+    },
+    attempts: {
+      type: 'integer',
+    },
+    status: {
+      type: 'keyword',
+    },
+    // NO NEED TO BE INDEXED
+    // traceparent: {
+    //   type: 'text',
+    // },
+    params: {
```

Comment:
`params` and `state` are JSON bags, so likely don't need to be indexed.

Reply:
I think it was failing test because we are using _source to get this attributes and since they are not mapped anymore, source was not working anymore.

Reply:
let me re-check/search and give you the right answer here

Reply:
ok You are right! we were able to not index these two, I tried multiple time and tests are not failing!